conf/distro/include/rdk-bpi.inc

```diff
@@ -21,3 +21,5 @@ DISTRO_FEATURES_append = " acl_nl_support"
 DISTRO_FEATURES_append = " referencepltfm "
 
 MACHINEOVERRIDES_append =. "${@bb.utils.contains('DISTRO_FEATURES', 'OneWifi', ':onewifi', '' ,d)}"
+
+DISTRO_FEATURES_append = " CPUPROCANALYZER_BROADBAND"
```

Comment:
can you confirm the test results in parity with - https://wiki.rdkcentral.com/display/RDK/RDK-B+CPUProcAnalyzer